usr/share/rear/layout/save/default/950_verify_disklayout_file.sh

```diff
@@ -198,14 +201,24 @@ for error_message in "${broken_part_errors[@]}" ; do
     LogPrintError "$error_message"
     disklayout_file_is_broken="yes"
 done
-for error_message in "${non_consecutive_part_errors[@]}" ; do
-    contains_visible_char "$error_message" || continue
-    LogPrintError "$error_message"
-    non_consecutive_partitions="yes"
-done
+
+#
+# Non consecutive partitions are now supported, unless parted < 2.0
+#
+local parted_version=$( get_version parted -v )
+test "$parted_version" || BugError "Function get_version could not detect parted version"
+if ! version_newer "$parted_version" 2.0 ; then
+    for error_message in "${non_consecutive_part_errors[@]}" ; do
+        contains_visible_char "$error_message" || continue
+        LogPrintError "$error_message"
+        non_consecutive_partitions="yes"
+    done
+    is_true "$non_consecutive_partitions" && Error "There are non consecutive partitions ('rear recover' would fail)"
```

Comment:
Strictly speaking the
```
is_true "$non_consecutive_partitions" && Error "There are non consecutive partitions ..."
```
line should stay at ist original place after the
```
is_true "$disklayout_file_is_broken" && BugError "Entries in $DISKLAYOUT_FILE are broken ..."
```
line because when both errors happen the `BugError` is more severe
so that it should error out with the `BugError` (and not with the `Error`).